bears/python/Flake8Bear.py

```diff
@@ -0,0 +1,23 @@
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+
+
+@linter(executable='flake8',
+        output_format='regex',
+        output_regex=r'(?P<filename>.*):(?P<line>\d+):(?P<column>\d+):\s'
+                     r'(?P<severity>[EFW]|C9|N8)[0-9]{2,3}\s'
+                     r'(?P<message>.*)',
+        severity_map={
+            "F": RESULT_SEVERITY.MAJOR,
+            "E": RESULT_SEVERITY.MAJOR,
+            "W": RESULT_SEVERITY.NORMAL,
+            "N8": RESULT_SEVERITY.INFO,
+            "C9": RESULT_SEVERITY.INFO})
+class Flake8Bear:
+    """
+    Checks for styling errors based on pep8, pyflakes, pycodestyle and mccabe
+    """
+
```

Comment:
Oh! I will fill them up. I was worrying why the tests are failing and totally forgot about it.